.github/workflows/publish-crates.yml

```diff
@@ -13,24 +17,42 @@ jobs:
     environment: "publish to crates.io"
     runs-on: ubuntu-latest
     steps:
-      - name: Install dependencies
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            override: true
+            profile: minimal
+      - name: Install build dependencies
         run: |
           wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
           echo "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-11 main" | sudo tee -a /etc/apt/sources.list
           sudo apt-get update
           sudo apt-get install -y clang-11 cmake
-      - name: Symlink libclang.so
-        run: sudo ln -s /lib/x86_64-linux-gnu/libclang-11.so.1 /lib/x86_64-linux-gnu/libclang.so
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
-        with:
-            toolchain: stable
-            override: true
-      - run: |
-          export crate_name=$(python3 -c "print('$GITHUB_REF'.split('/')[2].rsplit('_v', 1)[0])")
-          echo "Publishing crate: $crate_name"
-          cargo publish --locked --token ${CARGO_REGISTRY_TOKEN} --package "$crate_name"
+          # Create a symbolic link to ensure correct usage of libclang
+          sudo ln -s /lib/x86_64-linux-gnu/libclang-11.so.1 /lib/x86_64-linux-gnu/libclang.so
```

Comment:
Since the .so needs to be there, just possibly not in the location where it's being looked for, can this be fixed by adding the appropriate directory to the search path for the linker and/or setting LD_LIBRARY_PATH when whatever needs this library is being run? Adding symlinks to standard system locations seems like not a good idea for a build/test system to be doing.

Reply:
I see, after recheck. These lines seems not necessary.